### PR TITLE
openblas 0.3.24

### PR DESCRIPTION
**Pull request by p-linnane**
Created by `brew bump`

---

Created with `brew bump-formula-pr`.

### CODE REVIEW

**Comment by chenrui333** [comment]
```
../kernel/arm64/sgemm_kernel_sve_v2x8.S:1496:5: error: instruction requires: sve or sme
    whilelt p1.s, x9, x0
    ^
../kernel/arm64/sgemm_kernel_sve_v2x8.S:1497:5: error: instruction requires: sve or sme
    cntp x15, p0, p1.s
    ^
```

**Comment by fxcoudert** [comment]
Reported upstream: https://github.com/xianyi/OpenBLAS/issues/4212

**Review by fxcoudert — APPROVED** [review]

File: Formula/o/openblas.rb [hunk 1]
```diff
@@ -42,6 +42,9 @@ def install
     else
       Hardware.oldest_cpu.upcase.to_s
     end
+    # Apple chips do not support SVE
+    # https://github.com/xianyi/OpenBLAS/issues/4212
+    ENV["NO_SVE"] = "1"
```
Comment (SMillerDev) [inline Formula/o/openblas.rb:47]: How about the intel ones?

Comment (fxcoudert) [reply Formula/o/openblas.rb:47]: That's a NOP on Intel

**Review by fxcoudert — APPROVED** [review]

**Pull request merged**